tests/Microsoft.Identity.Test.Unit/ManagedIdentityTests/ImdsV2Tests.cs

```diff
@@ -130,5 +136,39 @@ public async Task GetCsrMetadataAsyncFails404WhichIsNonRetriableAndRetryPolicyIs
                 Assert.AreEqual(ManagedIdentitySource.DefaultToImds, miSource);
             }
         }
+
+        [TestMethod]
+        public void TestCsrGeneration()
+        {
+            var cuid = new CuidInfo
+            {
+                VmId = TestConstants.VmId,
+                VmssId = TestConstants.VmssId
```

Comment:
is there a test without vmssid?

Reply:
no, but I can remove from all and just make one vmId + vmssId test

Reply:
done